sbin/pfdns

```diff
@@ -379,6 +388,17 @@ sub regzone {
     return @answer;
 }
 
+sub add_answers_to_ip_port_pairs {
```

Comment:
I would suggest returning the pairs instead of doing it by reference.

Most of our code uses returns instead of passing things by reference

That would make the code clearer IMO